content/en/docs/concepts/containers/images.md

```diff
@@ -275,6 +275,8 @@ in private registries.
 {{< /note >}}
 
 Kubernetes supports specifying container image registry keys on a Pod.
+Each item in the `imagePullSecrets` array can only reference a Secret in the same namespace,
```

Comment:
This should move to the section below:

>  #### Referring to an imagePullSecrets on a Pod
> Now, you can create pods which reference that secret by adding an `imagePullSecrets`
> section to a Pod definition. <here>

I'd also consider rewording to:

```
`imagePullSecrets` must all be in the same namespace as the Pod. The referenced Secrets must be of type `kubernetes.io/dockercfg` or `kubernetes.io/dockerconfigjson`.
```

Otherwise LGTM from a technical POV.

Reply:
Thanks for the reviews, I have incorporated them.